3ds Max/Max2Babylon/Exporter/BabylonExporter.cs

```diff
@@ -767,6 +767,11 @@ public void Export(MaxExportParameters exportParameters)
             babylonScene.Prepare(false, false);
             if (isBabylonExported)
             {
+                if (!babylonScene.PackIndexArrays())
+                {
+                    RaiseWarning("Model has too many skeleton joints. The .babylon file will store joint indicies using expandanded form, this will only load properly using Babylon.js >= 8.5.0.");
```

Comment:
```suggestion
                    RaiseWarning("Model has too many skeleton joints. The .babylon file will store joint indices using expanded form, this will only load properly using Babylon.js >= 8.5.0.");
```